src/styles/card.scss

```diff
@@ -0,0 +1,140 @@
+@use './variables' as *;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  margin: 0 auto;
+
+  @media (min-width: 0) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  grid-auto-flow: column;
+  transition: 0.3s ease-in;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    text-align: center;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-family: $card-main-font;
+    font-weight: 500;
+    color: $card-text-main-color;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(120%);
+
+    .card__name {
+      transition: 0.3s ease-in;
+      color: #34568b;
+    }
+  }
+
+  &__description {
+    font-family: $card-main-font;
+    font-weight: $card-main-font-weight;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-text-secondary-color;
+    margin-bottom: $card-margin-main;
+  }
+
+  &__review {
+    font-family: $card-main-font;
+    font-weight: $card-main-font-weight;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-text-main-color;
+  }
+
+  &__price {
+    font-family: $card-main-font;
+    font-weight: $card-main-font-weight;
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-text-secondary-color;
+  }
+
+  &__amount {
+    font-family: $card-main-font;
+    font-size: 16px;
+    font-weight: 700;
+    color: $card-text-main-color;
+  }
+
+  &__wrapper-stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__wrapper-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: $card-margin-main;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-family: Roboto, sans-serif;
```

Comment:
The font-family should use the variable '$card-main-font' for consistency with the rest of the card styles.